README.rst

```diff
@@ -1,76 +1,40 @@
-|Build Status| |Codecov| |license|
+|Build Status| |npm_version| |license| |semantic-release|
 
-frontend-template-application
-=================================
-
-Please tag **@edx/fedx-team** on any PRs or issues.  Thanks.
+frontend-app-report-viewer
+==========================
 
 Introduction
 ------------
 
-This repository is a template for Open edX micro-frontend applications. It is flagged as a Template Repository, meaning it can be used as a basis for new GitHub repositories by clicking the green "Use this template" button above.  The rest of this document describes how to work with your new micro-frontend after you've created a new repository from the template.
-
-After Copying The Template
---------------------------
-
-You'll want to do a find-and-replace to replace all instances of ``frontend-template-application`` with the name of your new repository.  Also edit index.html to replace "Application Template" with a friendly name for this application that users will see in their browser tab.
-
-**Prerequisite**
-
-`Devstack <https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/installation/index.html>`_.  If you start Devstack with ``make dev.up.ecommerce`` that should give you everything you need as a companion to this frontend.
-
-**Installation and Startup**
-
-In the following steps, replace "frontend-template-application' with the name of the repo you created when copying this template above.
-
-1. Clone your new repo:
-
-  ``git clone https://github.com/edx/frontend-template-application.git``
-
-2. Use node v12.x.
-
-   The micro-frontend build scripts support node 12.  Using other major versions of node *may* work, but is unsupported.  For convenience, this repository includes an .nvmrc file to help in setting the correct node version via `nvm <https://github.com/nvm-sh/nvm>`_.
-
-3. Install npm dependencies:
-
-  ``cd frontend-template-application && npm install``
-
-4. Update the application port to use for local development:
+This MFE allows an instructor to view all the reports they have access to in the browser,
+in an in-browser CSV Viewer.
```

Comment:
I think this can be made simpler. For example.

```
This MFE allows an instructor to view all the reports they have access to in an in-browser CSV Viewer.
```